README.md

```diff
@@ -114,6 +114,17 @@ those defined in `etc/defaults.conf`, set them on `etc/conf` instead i.e:
     $ echo 'XBPS_CFLAGS="your flags here"' >> etc/conf
     $ echo 'XBPS_LDFLAGS="your flags here"' >> etc/conf
 
+Some common options:
+
+- `XBPS_CFLAGS`, `XBPS_CXXFLAGS`, `XBPS_LDFLAGS`: define common compiler flags
+   to use when building packages.
+- `XBPS_ALLOW_RESTRICTED`: allow xbps-src to build packages that are flagged
+   as restricted. This is disabled by default.
+- `XBPS_CHECK_PKGS`: run the (optional) `do_check()` function of packages which
+   runs tests on the package after building. This is disabled by default.
+- `XBPS_MAKEJOBS`: allow parallel builds of packages that use make or similar
```

Comment:
IMHO they are already explained enough in etc/defaults.conf

Reply:
If that's where documentation should live, maybe we could mention that it exists there then?